recipes/baked_beans.json

```diff
@@ -0,0 +1,18 @@
+{
+    "recipe_name": "Baked Beans",
+    "serves": "1",
```

Comment:
This is a bit sad, maybe make it 2 and include more beans?

Reply:
Two tins of beans for two people?

Reply:
Done